flake.nix

```diff
@@ -28,6 +28,10 @@
             overlays = [
               # Load in various overrides for custom packages and version pinning.
               (import ./ci/overlay.nix { pkgs = pkgs; })
+              (final: prev: {
```

Comment:
This is being done in the `overlay.nix` for a bunch of other packages. Would you mind moving this block into the `./ci/overlay.nix`?